src/main/kotlin/com/terraformation/backend/seedbank/model/SeedQuantity.kt

```diff
@@ -42,8 +42,11 @@ fun SeedQuantityUnits.fromGrams(quantity: BigDecimal): BigDecimal {
   return value.stripTrailingZeros()
 }
 
-data class SeedQuantityModel(val quantity: BigDecimal, val units: SeedQuantityUnits) :
-    Comparable<SeedQuantityModel> {
+data class SeedQuantityModel(
+    val quantity: BigDecimal,
+    val units: SeedQuantityUnits,
+    val notes: String? = null,
```

Comment:
This also feels like the wrong place for notes; a quantity is more like a numeric value and it doesn't seem to me like there could be reasonable semantics for notes when, e.g., you subtract one quantity from another.